notecard/notecard.go

```diff
@@ -237,9 +238,15 @@ func OpenI2C(port string, portConfig int) (context Context, err error) {
 	context.PortEnumFn = i2cPortEnum
 	context.PortDefaultsFn = i2cDefault
 	context.CloseFn = cardCloseI2C
+	context.ReopenFn = cardReopenI2C
 	context.ResetFn = cardResetI2C
 	context.TransactionFn = cardTransactionI2C
 
+	// Record I2C configuration
+	context.isSerial = false
+	context.i2cName = port
+	context.i2cAddress = portConfig
+
 	// Open the I2C port
```

Comment:
Here's where we stash I2C context. Mostly academic at this point since I haven't implemented Reopen() for I2C yet. I don't have a system on which I could easily test that and I don't need it right now, but there's a placeholder below.